Source/WebCore/inspector/InspectorCanvas.cpp

```diff
@@ -803,14 +783,6 @@ void InspectorCanvas::recordAction(String&& name, InspectorCanvasCallTracer::Pro
 
     if (is<ImageBitmapRenderingContext>(context) && shouldSnapshotBitmapRendererAction(name))
         m_contentChanged = true;
-#if ENABLE(WEBGL)
-    else if (is<WebGLRenderingContext>(context) && shouldSnapshotWebGLAction(name))
-        m_contentChanged = true;
```

Comment:
are we sure that `CanvasObserver::canvasChanged` will _always_ be called whenever the `<canvas>` changes _and_ in a timely manner?  my concern is that without reading the contents after these actions maybe they might not actually get drawn until later.  perhaps i am also just totally not remembering what little WebGL i knew from many years ago 😅

as another option, maybe instead of removing this we could check and bail if we're about to draw to a buffer instead of the context?